### DISCUSSION TITLE
Exporting straight from Glyphs.app vs. building with toolchain?

### DISCUSSION THREAD

**Discussion by jakubfoglar** [rsms/inter#734] [Q&A] [answered]
Hi! I’m in the process of modifying some characters of Inter 4 to suit a project.

I’m wondering if it’s enough to export the fonts from Glyphs, or if I should be going through the whole toolchain to build the fonts? Is there some important functionality that the fonts will not have if not built with the toolchain? (i.e. some special hinting,...)

Thanks!

**Comment by rsms** [comment, marked as answer]
In theory it should work to export from Glyphs. You can look at the Makefile in the source repository to see what is being done. It’s mainly a bunch of metadata fixes. In particular the variable font may not work as expected if exported from Glyphs.

I recommend using the Makefile. You should be able to run it on a Mac, BSD or Linux system (or WSL on Windows.)

**Reply by jakubfoglar** [reply]
I went down the rabbit hole in the mean time and ended up not using most of the processes and even weight ranges and language sets, for this particular project. So I am building with Glyphs with a fraction of the original project. Thanks Rasmus!
